DemoPage/examples/LoadingProgress/index.js

```diff
@@ -1,7 +1,7 @@
-"use strict";
+'use strict';
 
 var React                = require('react');
-var LoadingProgress      = require('../../../Helpers/LoadingProgress');
+var LoadingProgress      = require('../../../atoms/LoadingProgress');
```

Comment:
It's a thing, as like "these are small pieces, which cannot be used to nest smaller pieces, but which you use to build bigger stuff". Helpers describes something completely different, like, "these things make other things easier or more convenient".